compute/admin_guide/alerts/alert_mechanism.adoc

```diff
@@ -1,98 +1,76 @@
 == Alert mechanism
 
-Prisma Cloud lets you surface critical policy breaches by sending alerts to any number of channels.
-Alerts ensure that significant events are put in front of the right audience at the right time.
+Prisma Cloud generates alerts to help you focus on the significant events that need your attention. 
+Because alerts surface policy violations, you need to put it in front of the right audience and in a timely manner. 
+To meet this need, you can create alert profiles that send events/notifications to the channels your internal teams use to triage and address these violations.
```

Comment:
I think we should replace "channels" with "Alert providers' across all the document